docs/deploy.rst

```diff
@@ -96,6 +112,9 @@ exodus-pipeline.yaml, run by the `AWS CLI`_ command
             repoOwner=... repoName=... repoBranch=... \
             githubToken=... region=... email=... \
             project=... useCloudTrail=... \
```

Comment:
the `useCloudTrail` option should be removed from the exodus-pipeline deployment

Reply:
+1